pkg/hub/localizer/localizer.go

```diff
@@ -32,6 +37,12 @@ import (
 	applisters "github.com/clusternet/clusternet/pkg/generated/listers/apps/v1alpha1"
 	"github.com/clusternet/clusternet/pkg/known"
 	"github.com/clusternet/clusternet/pkg/utils"
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/ghodss/yaml"
+)
+
+var (
+	ErrMissingValue = errors.New("missing value")
```

Comment:
This var is used only once. No need to declare it.

Reply:
dome